scenes/game/local_game.gd

```diff
@@ -3044,6 +3044,8 @@ func is_effect_condition_met(performing_player : Player, effect, local_condition
 			return performing_player.is_at_edge_of_arena()
 		elif condition == "boost_in_play":
 			return performing_player.continuous_boosts.size() > 0
+		elif condition == "no_boost_in_play":
+			return performing_player.continuous_boosts.size() < 0			
```

Comment:
it can't be less than 0.
should be == 0 or <= 0, but really an array size can't be less than 0.

Reply:
gave it the ==